### PR TITLE
Ec explain

### PR DESCRIPTION
**Pull request by wiktormowinski**

### CODE REVIEW

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# EC power states in NovaCustom laptops
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:1]: let's make this a bit more generic, I suspect we'll be adding more sections here sooner or later

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Way the EC works right now
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:3]:
```suggestion
## Power states

This section describes the logic used by the EC to control platform power states
depending on AC adapter presence.
```

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+EC disables all power planes, and the platform is set to G3 power state.
+Therefore, in that case pressing the power button results in a coldboot.
+(S0 → S5 → G3 → S0)
+
+### Turning the computer off while not plugged in
+
+The machine actually enters S5 briefly and immediately after, it switches
+to G3 like intended. Like mentioned before, turning the computer from G3
+results in a coldboot. (S0 → S5 → G3 → S0 )
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:21]:
```suggestion
The machine actually enters S5 and proceeds to G3 soon after. Pressing the
power button will result a coldboot. (S0 → S5 → G3 → S0 )
```

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Way the EC works right now
+
+### Turning off the computer while charger is plugged in
+
+The machine switches to system power state S5, using little to no
+power. As a result pressing the power button to turn it on, will result in
+a warmboot. (S0 → S5 → S0)
+
+### Cutting off power feed while in S5
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:11]:
```suggestion
### AC detach while turned off
```

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# EC power states in NovaCustom laptops
+
+## Way the EC works right now
+
+### Turning off the computer while charger is plugged in
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:5]:
```suggestion
### Shutdown, with AC connected
```

File: docs/transparent-validation/ec-information/ec-explained.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+power. As a result pressing the power button to turn it on, will result in
+a warmboot. (S0 → S5 → S0)
+
+### Cutting off power feed while in S5
+
+EC disables all power planes, and the platform is set to G3 power state.
+Therefore, in that case pressing the power button results in a coldboot.
+(S0 → S5 → G3 → S0)
+
+### Turning the computer off while not plugged in
```
Comment (mkopec) [inline docs/transparent-validation/ec-information/ec-explained.md:17]:
```suggestion
### Shutdown, with AC disconnected
```

**Review by mkopec — CHANGES_REQUESTED** [review]

**Comment by wiktormowinski** [comment]
> I guess you also need to add this file to mkdocs.yml

yeah, I forgot to push 2nd commit, mb: https://github.com/Dasharo/docs/pull/904/commits/576812772a1d950b53b6a25ede409a1a03e7f991

**Review by mkopec — APPROVED** [review]

**Pull request merged**